src/lib/share/index.js

```diff
@@ -19,6 +19,16 @@ export const isSharingAvailable = Platform.select({
   default: isMobileNative, // ios or android
 })
 
+/**
+ * Represents all of the metadata needed by a vendor to complete a linked transactions
+ */
+export interface VendorMetadata {
+  callbackUrl: URL;
+  invoiceData: string;
+  website: URL;
+  vendorName: string;
```

Comment:
we should probably minimize the field names also here, so they dont take much space on the base64 encryption